kubernetes/samples/scripts/domain-lifecycle/clusterStatus.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+# !/bin/sh
```

Comment:
Has it been confirmed taht this script works with 'sh' instead of 'bash'?

Reply:
yes, the script works with sh. All the scripts in this directory has the inclusion of "# !/bin/sh"